packages/app/src/templates/components/delete-confirmation-modal/delete-confirmation-modal.component.tsx

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
I'm not sure if this is a proper way to display that kind of notification. Let's re-think within `notification` functionality. It will more generic

Reply:
From my point of view, this is a notification about deleting the template. You can create a notification with the type `popup` and then remove it with delay.